mm2src/derives/enum_from/src/lib.rs

```diff
@@ -84,19 +86,96 @@ pub fn enum_from_trait(input: TokenStream) -> TokenStream {
     }
 }
 
+/// `EnumFromStringify` is very useful for generating `From<T>` trait from one enum to another enum
+/// currently, this crate can only convert enum variant with only some basic inner type such as `String`, and `Enum`
+/// type just like the example below. Can not be used for tuple, struct etc for now .
+/// More support will be added.
+///
+///
+/// ### USAGE:
+/// ```rust
+/// use enum_from::EnumFromStringify;
+/// use std::fmt::{Display, Formatter};
+///
+/// // E.G, this converts from whatever Bar is to FooBar::Bar(String) and
+/// // whatever Foo to FooBar::Foo(Foo)
+/// #[derive(Debug, EnumFromStringify, PartialEq, Eq)]
+/// pub enum FooBar {
+///     #[from_stringify("Bar", "String", "Man")]
+///     Bar(String),
```

Comment:
There is already `EnumFromInner::from_inner`:
```rust
#[derive(EnumFromInner)]
enum FooBar {
  #[from_inner(String)]
  Bar(String),
}
```
that expands to
```rust
impl From<String> for FooBar {
  fn from(s: String) -> FooBar { FooBar::Bar(s) }
}
```

Reply:
ok 👍🏼